File: safari-600.7-branch/LayoutTests/http/tests/uri/resources/print-uri.php
<?php
    header("Content-Type: text/javascript");
    header("Expires: Thu, 01 Dec 2003 16:00:00 GMT\n");
    header("Cache-Control: no-store, no-cache, must-revalidate\n");
    header("Pragma: no-cache\n");
    echo "document.write('" . $_SERVER['REQUEST_URI'] . "');";
?>

File: LayoutTests/http/tests/appcache/resources/print-uri.py
#!/usr/bin/env python3

import os
import sys

uri = os.environ.get('REQUEST_URI', '')

sys.stdout.write(
    'Content-Type: text/javascript\r\n'
    'Expires: Thu, 01 Dec 2003 16:00:00 GMT\r\n'
    'Cache-Control: no-cache, must-revalidate\r\n'
    'Pragma: no-cache\r\n\r\n'
    f'{uri}'
)